op-node/rollup/driver/state.go

```diff
@@ -426,34 +427,22 @@ func (s *Driver) syncStatus() *eth.SyncStatus {
 	}
 }
 
-// SyncStatus blocks the driver event loop and captures the syncing status.
-// If the event loop is too busy and the context expires, a context error is returned.
+// updateSyncStatusCache updates the cached sync status
+func (s *Driver) updateSyncStatusCache() {
+	ss := s.syncStatus()
+	s.syncStatusCache.Store(ss)
+}
+
+// SyncStatus returns the syncing status.
 func (s *Driver) SyncStatus(ctx context.Context) (*eth.SyncStatus, error) {
```

Comment:
what happens if the returned sync status is nil because it has not been initialized yet?

Reply:
It returns a `null` JSON object, which I think is what we want. Seems less misleading than a zero valued `SyncStatus` struct.